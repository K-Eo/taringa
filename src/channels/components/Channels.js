import React from 'react'

import Ad from '../../ads/components/Ad'
import Filterable from '../../filters/components/Filterable'

const Channels = () => {
  const filters = {
    hot: {
      displayName: 'Destacados',
      exact: true,
      id: 'hot',
      pathname: '/channels',
      url: '/c/list/hot',
    },
    recents: {
      displayName: 'Recientes',
      exact: false,
      id: 'recents',
      pathname: '/channels/recents',
      url: '/c/list/recent',
    },
    populars: {
      displayName: 'Populares',
      exact: false,
      id: 'tops',
      pathname: '/channels/populars',
      url: '/c/list/popular',
    },
  }

  return (
    <div className="row">
      <Filterable filters={filters} />

      <div className="col-8" />

      <div className="col-4">
        <Ad />
      </div>
    </div>
  )
}

export default Channels
