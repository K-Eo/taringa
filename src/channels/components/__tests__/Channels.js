import React from 'react'

import Channels from '../Channels'

describe('Channels', () => {
  it('exists', () => {
    expect(Channels).toBeDefined()
  })

  it('renders', () => {
    const wrapper = shallow(<Channels />)

    expect(wrapper).toMatchSnapshot()
  })
})
