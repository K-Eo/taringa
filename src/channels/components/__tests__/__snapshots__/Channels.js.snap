// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Channels renders 1`] = `
<div
  className="row"
>
  <Connect(Filterable)
    filters={
      Object {
        "hot": Object {
          "displayName": "Destacados",
          "exact": true,
          "id": "hot",
          "pathname": "/channels",
          "url": "/c/list/hot",
        },
        "populars": Object {
          "displayName": "Populares",
          "exact": false,
          "id": "tops",
          "pathname": "/channels/populars",
          "url": "/c/list/popular",
        },
        "recents": Object {
          "displayName": "Recientes",
          "exact": false,
          "id": "recents",
          "pathname": "/channels/recents",
          "url": "/c/list/recent",
        },
      }
    }
  />
  <div
    className="col-8"
  />
  <div
    className="col-4"
  >
    <Ad />
  </div>
</div>
`;
